routes/problems.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const datastore = require('../modules/datastore.js');
+
+const PROBLEMS_KIND = "Problem";
+
+/**
+ * @typedef {Object} Problem
+ * @param {number} id
+ * @param {string} title
+ * @param {string} text
+ */
+class Problem {
+  constructor(id, title, text) {
+    this.id = id;
+    this.title = title;
+    this.text = text;
+  }
+}
+
+module.exports = function(app) {
+  app.get('/problems', async function(request, response) {
+    const query = datastore.createQuery(PROBLEMS_KIND);
+    const [entities] = await datastore.runQuery(query);
```

Comment:
same comment here as for Nick's pull request - pagination is important when the API may return multiple pages.

Reply:
Right now, this route just queries for all problems. I added a todo to add a limit to the query and support pagination post-MVP.